.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  Ubuntu:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+    - name: Prepare
+      run: sudo apt update && sudo apt install -y unzip wget
+    - name: Download and extract Partiels binary
+      run: |
+        TAG=$(curl -s https://api.github.com/repos/Ircam-Partiels/Partiels/releases | jq -r '.[0].tag_name')
+        wget https://github.com/Ircam-Partiels/Partiels/releases/download/$TAG/Partiels-Linux.tar.gz
+        tar -xzf Partiels-Linux.tar.gz
+        sh Partiels/Partiels-install.sh
+        Partiels/Partiels --help
+    - name: Install dependencies
+      run: pip install pytest
+    - name: Run tests
+      run: pytest tests/
+  
+  Windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Download and extract Partiels binary
+        shell: powershell
+        run: |
+          $ErrorActionPreference = "Stop"
+          $response = Invoke-RestMethod -Uri "https://api.github.com/repos/Ircam-Partiels/Partiels/releases"
+          $tag = $response[0].tag_name
+          $url = "https://github.com/Ircam-Partiels/Partiels/releases/download/$tag/Partiels-Windows.exe"
+          Invoke-WebRequest -Uri $url -OutFile "Partiels.exe"
+          & .\Partiels.exe --help
+      - name: Install dependencies
+        run: pip install pytest
+      - name: Run tests
+        run: pytest tests/
+  
+  MacOS:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Download Partiels DMG
+        shell: bash
+        run: |
+          JSON=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/Ircam-Partiels/Partiels/releases)          
+          TAG=$(echo "$JSON" | jq -r '.[0].tag_name')
+          URL="https://github.com/Ircam-Partiels/Partiels/releases/download/$TAG/Partiels-MacOS.dmg"
+          curl -L -v -o Partiels.dmg "$URL"
+      - name: Mount Partiels DMG
+        shell: bash
+        run: |
+          MOUNT_DIR=$(hdiutil attach Partiels.dmg | grep Volumes | awk '{print $3}')
+          cp -R "$MOUNT_DIR/Partiels.app" /Applications/
+          "/Applications/Partiels.app/Contents/MacOS/Partiels" --help
+          echo "$MOUNT_DIR" > mount_dir.txt
+      - name: Install dependencies
+        run: pip install pytest
+      - name: Run tests
+        run: pytest tests/
+      - name: Unmount Partiels DMG
```

Comment:
I think this will be managed automatically, otherwise it should be placed in `Mount Partiels DMG` which can be renamed Install Partiels?